src/website/src/app/documentation/demos/header/header-types.demo.html

```diff
@@ -53,8 +53,8 @@
             <div class="header-actions">
                 <clr-dropdown>
                     <button class="nav-icon" clrDropdownTrigger>
```

Comment:
the label should go on the button, as @gnomeontherun says.

but it would be even better if it programmatically updated the label to something like...

`<button class="nav-icon" clrDropdownTrigger [attr.aria-label]="isOpen ? 'close settings menu' : 'open settings menu'">`

Reply:
`isOpen` is hard to get and will require more code for every dropdown out-there. I think the idea of passing `label` values to the `clrDropdownTrigger` is much more "working out of the box"

Reply:
No `title`. We can also use 'toggle settings menu' perhaps. This is just our documentation and we want to show that we're doing the right stuff in our docs so apps adopt it too.